### DISCUSSION TITLE
Do I need to manually download and install compatibility patches?

### DISCUSSION THREAD

**Discussion by Ivashamagnus** [CombatExtended-Continued/CombatExtended#2541] [General]
How to install compatibility patches correctly?

**Comment by Ivashamagnus** [comment]
How to download? Need direct access to repositories?

**Comment by SamaelGray** [comment]
Nope, the compatibility patches are built-into CE, you can either subscribe to CE's steam workshop item or download the latest dev snapshot of CE linked in the description of the mod in the main repo page.

**Reply by Ivashamagnus** [reply]
Thx!!!
